core/templates/services/rte-helper-modal.controller.spec.ts

```diff
@@ -16,21 +16,73 @@
  * @fileoverview Unit tests for RteHelperModalController.
```

Comment:
Some tests were removed in the interest of getting this PR in earlier. An issue was filed for adding the tests back. #18390. Checked this with @seanlip.